apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx

```diff
@@ -357,7 +357,7 @@ const ColumnEditor = ({
           <Input
             label="CHECK Constraint"
             labelOptional="Optional"
-            placeholder={`e.g length(${columnFields?.name || 'column_name'}) < 500`}
+            placeholder={`e.g length(${columnFields?.name || 'column_name'}) > 0`}
```

Comment:
This is not what I meant.

The type of the field is `int4`.
I want the placeholder to be something like:
- `duration_min > 0`
- `duration_min < 0`
- `duration_min < -100` (doesn't make sense but it's ok because we don't understand the semantic of the field name)

I don't meant the placeholder should be changed to `> 0` to fit the meaning of my table's field's name.

I just think that checking the length of an integer doesn't make sense.